templates/mcq.jsx

```diff
@@ -0,0 +1,104 @@
+import Adapt from 'core/js/adapt';
+import { templates, classes, html, compile } from 'core/js/reactHelpers';
+
+export default function(model, view) {
+  const data = model.toJSON();
+  const ariaLabels = Adapt.course.get('_globals')._accessibility._ariaLabels;
+
+  const isInteractive = model.isInteractive();
+
+  return (
+    <div className='component__inner mcq__inner'>
+
+      {templates.component(model, view)}
+
+      <div
+        className={classes([
+          'component__widget',
+          'mcq__widget',
+          !data._isEnabled && 'is-disabled',
+          data._isInteractionComplete && 'is-complete is-submitted show-user-answer',
+          data._isCorrect && 'is-correct'
+        ])}
+        role={data._isRadio ? 'radiogroup' : 'group'}
+      >
+
+        {data._items.map(({ text, _index, _isActive, _shouldBeSelected, _isHighlighted }, index) =>
+
+          <div
+            className={classes([
+              `mcq__item item-${index}`,
+              (!isInteractive && data._canShowMarking && _shouldBeSelected) ? 'is-correct' : null,
+              (!isInteractive && data._canShowMarking && !_shouldBeSelected) ? 'is-incorrect' : null
+            ])}
+            key={_index}
+          >
+
+            <input
+              className='mcq__item-input'
+              id={`${data._id}-${index}-input`}
+              name={data._isRadio ? `${data._id}-item` : null}
+              type={data._isRadio ? 'radio' : 'checkbox'}
+              disabled={!data._isEnabled}
+              aria-label={(isInteractive || !data._canShowMarking) ?
+                Adapt.a11y.normalize(text) :
+                `${_shouldBeSelected ? ariaLabels.correct : ariaLabels.incorrect}, ${_isActive ? ariaLabels.selectedAnswer : ariaLabels.unselectedAnswer}. ${Adapt.a11y.normalize(text)}`}
```

Comment:
As usual, comes down to personal preference but selecting by bracket notation is an alternative with less repetition:
```js
`${ariaLabels[_shouldBeSelected ? 'correct' : 'incorrect']}, ${ariaLabels[_isActive ? 'selectedAnswer' : 'unselectedAnswer']}. ${Adapt.a11y.normalize(text)}`}
```

Reply:
You no like this?